maintenance_request_time_performance/hooks.py

```diff
@@ -0,0 +1,44 @@
+from odoo import _, api, SUPERUSER_ID
+from datetime import datetime, timedelta
+
+
+def post_init_hook(cr, registry):
+    with api.Environment.manage():
+        env = api.Environment(cr, SUPERUSER_ID, {})
+        maintenance_request = env['maintenance.request']
```

Comment:
It seems there's an Odoo/OCA convention with model object naming: if the model is `maintenance.request`, an object for the model itself (not for a record set) should be named `MaintenanceRequest`. So, I'd revert this change...

Reply:
ok. I changed because Pycharm was giving me a warning "Variable in function shoud be lowercase". I'll revert this change.